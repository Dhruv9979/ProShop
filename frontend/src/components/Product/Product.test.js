import React from 'react';
import { shallow } from 'enzyme';
import Product from './Product.component';

describe('Product component', () => {
    let wrapper;
    beforeEach(() => {
        const mockProduct = {
            _id: '1',
            name: 'Test Product',
            image: '/images/test.jpg',
            description:
              'Testing the Test product',
            brand: 'ProShop',
            category: 'Electronics',
            price: 99.99,
            countInStock: 10,
            rating: 4.5,
            numReviews: 12,
          }

          wrapper = shallow(<Product product={mockProduct} />)
    });
    it('should render Product component', () => {
        expect(wrapper).toMatchSnapshot();
    });
});