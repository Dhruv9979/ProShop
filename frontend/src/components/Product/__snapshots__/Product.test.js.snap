// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Product component should render Product component 1`] = `ShallowWrapper {}`;
